test/e2e/apps/rc.go

```diff
@@ -84,6 +97,175 @@ var _ = SIGDescribe("ReplicationController", func() {
 	framework.ConformanceIt("should release no longer matching pods", func() {
 		testRCReleaseControlledNotMatching(f)
 	})
+
+	ginkgo.It("should test the lifecycle of a ReplicationController", func() {
+		testRcName := "rc-test"
+		testRcNamespace := ns
+		testRcInitialReplicaCount := int32(1)
+		testRcMaxReplicaCount := int32(2)
+		rcResource := schema.GroupVersionResource{Group: "", Version: "v1", Resource: "replicationcontrollers"}
+
+		rcTest := v1.ReplicationController{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:   testRcName,
+				Labels: map[string]string{"test-rc-static": "true"},
+			},
+			Spec: v1.ReplicationControllerSpec{
+				Replicas: &testRcInitialReplicaCount,
+				Selector: map[string]string{"test-rc-static": "true"},
+				Template: &v1.PodTemplateSpec{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:   testRcName,
+						Labels: map[string]string{"test-rc-static": "true"},
+					},
+					Spec: v1.PodSpec{
+						Containers: []v1.Container{{
+							Name:  testRcName,
+							Image: "nginx",
+						}},
+					},
+				},
+			},
+		}
+
+		ginkgo.By("creating a ReplicationController")
+		// Create a ReplicationController
+		_, err := f.ClientSet.CoreV1().ReplicationControllers(testRcNamespace).Create(context.TODO(), &rcTest, metav1.CreateOptions{})
+		framework.ExpectNoError(err, "Failed to create ReplicationController")
+
+		// setup a watch for the RC
+		rcWatchTimeoutSeconds := int64(180)
```

Comment:
Nit that can be addressed in followup PR: this is a magic number that could use some explanation.  Why was three minutes chosen?

Reply:
It seemed to fail on shorter times due to other tests, I believe.
There was another test where in last week's meeting [this was discussed](https://github.com/kubernetes/kubernetes/pull/88778#issuecomment-603470581).

> @BobyMCbobs 180 seconds was chosen was initially at or below 60, it seemed.. I changed it to make it unflakey. Happy to change it.